onnx/test/shape_inference_test.py

```diff
@@ -380,15 +432,22 @@ def test_matmul_allow_unknown(self, _, version) -> None:
 
     @parameterized.expand(all_versions_for("Cast"))
     def test_cast(self, _, version) -> None:
+        if version == 1:
+            target_type = "UINT8"
+        else:
+            target_type = TensorProto.UINT8  # type: ignore[assignment]
         graph = self._make_graph(
             [("x", TensorProto.FLOAT, (2, 4, 3))],
-            [make_node("Cast", ["x"], ["y"], to=TensorProto.UINT8)],
+            [make_node("Cast", ["x"], ["y"], to=target_type)],
             [],
+            version,
         )
         self._assert_inferred(
             graph,
             [make_tensor_value_info("y", TensorProto.UINT8, (2, 4, 3))],
             opset_imports=[helper.make_opsetid(ONNX_DOMAIN, version)],
+            # Concat Version 1 does not have shape inference function.
```

Comment:
Do you mean Cast?

Reply:
Good catch... Fixed.
Sorry for the late response.